src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirBold;
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: 300;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.field {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  color: rgba(61, 78, 97, 1);
+  display: block;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  padding-top: 22px;
+  padding-bottom: 23px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.field:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.field:focus {
+  font-family: AvenirBold, sans-serif;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.field::placeholder {
+  font-family: Avenir, sans-serif ;
+  color: rgba(61, 78, 97, 1);
+  width: 129px;
+  height: 22px;
+}
+
+.icon {
+  position: absolute;
+  top: 45px;
+  left: 33px;
+}
+
+.field--small {
+  box-sizing: border-box;
+  width: 100%;
+  height: 42px;
+  font-family: Avenir, sans-serif;
+  font-size: 14px;
+  color: rgba(61, 78, 97, 1);
+  display: block;
+  padding-left: 33px;
+  padding-top: 9px;
+  padding-bottom: 10px;
+  padding-right: 226.71px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.field--small:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.field--small:focus {
+  font-family: AvenirBold, sans-serif;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.field--small::placeholder {
+  font-family: Avenir, sans-serif;
+  color: rgba(61, 78, 97, 1);
+  width: 167.9px;
+  height: 20px;
+}
```

Comment:
Here are still a lot of duplicates. These styles are duplicates of bigSearchInput, so create a class `searchInpunt` and move there all connected styles.

Check examples in [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 2nd item

Feel free to ask me in the private channel if you don't have ideas how to fix this